src/routes/auth.go

```diff
@@ -55,21 +55,19 @@ func AuthRoutes(reg *gin.RouterGroup, db *bun.DB) {
 
 	googleAuth.GET("/login", func(ctx *gin.Context) {
 		url := conf.AuthCodeURL("state")
-		fmt.Printf("Visit the URL for the auth dialog: %v", url)
-
 		ctx.Redirect(302, url)
 	})
 
 	googleAuth.GET(("/callback"), func(ctx *gin.Context) {
 		code := ctx.Query("code")
```

Comment:
*not needed now
but maybe let's start thinking in terms of creating controllers instead of writing the logic in the routes file, what do you think about this?

Reply:
- yes @ivinayakg need to make this change